app/src/main/java/com/quran/labs/androidquran/util/AudioUtils.kt

```diff
@@ -75,6 +75,16 @@ constructor(private val quranInfo: QuranInfo, private val quranFileUtils: QuranF
     return if (rootDirectory == null) null else rootDirectory + item.path
   }
 
+  fun getLocalQariUri(context: Context, item: QariItem): String? {
```

Comment:
why not just use `getQariUrl` in this same class?

Reply:
I actually used that initially but found that the method uses `"%03d%03d"` but instead it should be `"%03d" + File.separator + "%03d"`. So I wasn't sure if I could change the existing `getQariUrl` method without breaking existing code so I skipped it and wrote a new function altogether. Would you like me to use `getQariUrl` and make the necessary modifications so that it works correctly for gapless?

Reply:
ohh actually, `getQariUrl` gives me the remote URL, I want a local URL so that I can find the file and delete. So please ignore my above comment